protocol/lavasession/single_provider_session.go

```diff
@@ -159,6 +159,30 @@ func (sps *SingleProviderSession) PrepareSessionForUsage(ctx context.Context, cu
 		// reading userSessionParent address because it's a fixed string value that isn't changing
 		utils.LavaFormatWarning("CU Mismatch within the threshold", nil, utils.Attribute{Key: "GUID", Value: ctx}, utils.Attribute{Key: "missingCU", Value: missingCU}, utils.Attribute{Key: "consumer", Value: sps.userSessionsParent.consumerAddr},
 			utils.Attribute{Key: "sessionID", Value: sps.SessionID}, utils.Attribute{Key: "relayNum", Value: sps.RelayNum})
+
+		// Badge Session: verify there are enough missing cus allowed
+		if badgeUser != "" {
```

Comment:
based on the comment before, remove this section

Reply:
Missing CU removed from badge in this commit: 68504da